google-cloud-clients/google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITStorageTest.java

```diff
@@ -162,6 +162,7 @@
   public static void beforeClass() throws IOException {
     remoteStorageHelper = RemoteStorageHelper.create();
     storage = remoteStorageHelper.getOptions().getService();
+
```

Comment:
I'm assuming you're adding integration tests (I think you mentioned it), but I want to make sure they are still added before this is merged.